ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -178,6 +179,15 @@ RnsShell::LayerInvalidateMask RSkComponentImage::updateComponentProps(const Shad
       imageProps.tintColor = RSkColorFromSharedColor(newimageProps.tintColor,SK_ColorTRANSPARENT);
     }
     if((forceUpdate) || (oldimageProps.sources[0].uri.compare(newimageProps.sources[0].uri) != 0)) {
+      if( isRequestInProgress_ && remoteCurlRequest_){
+        auto sharedCurlNetworking = CurlNetworking::sharedCurlNetworking();
+        // if url is changed, image component is get component property update.
+        // calncel the onging request and made new request to network.  
+        sharedCurlNetworking->abortRequest(remoteCurlRequest_);
+        remoteCurlRequest_ = nullptr;
+        //TODO - need to send the onEnd event to APP if it is abort.
```

Comment:
Regarding the point --  //TODO - need to send the onEnd event to APP if it is abort.
HAve we confirmed with reference On ENd to be sent ??
If not, can we change the TODO need to send the onEnd event to APP if it is abort to be checked & confirmed with reference